platform/node/scripts/after_success.sh

```diff
@@ -3,10 +3,13 @@
 set -e
 set -o pipefail
 
-if [[ "${PUBLISH:-}" == "true" ]]; then
-    if [[ "${BUILDTYPE}" == "Release" ]]; then
+if [[ -n ${PUBLISH:-} ]]; then
```

Comment:
This looks good - but I want to mention that I think it would work just the same without the `-n`.